### PR TITLE
Misc Fixes

### PR DESCRIPTION
**Pull request by Arch881010**
- [x] Wrote some decent code.
- [ ] Tested such code.

Changelist:
Commit # 1: Set up basics of formatting numbers. (Not been tested -- careful.)
Commit # 2: Add minutes/hours/days before the game start time.
Commit # 3: Fix Commit # 1. 

Review and make sure it looks right for Commit #s 3 and 1.

### CODE REVIEW

File: src/commands/playercount.ts [hunk 1]
```diff
@@ -17,7 +17,25 @@ const playerCount = async (options: CommandExecuteParameters) => {
...
+
+  // EG: "There are currently **1 operatives online.**" -> "**There is currently 1 operative online.**"
+  if(Number(playerCount.Online_players) == 1) operativesText = "**There is currently 1 operative online.**"; // If (1 == 1), should also = 1.
+  // EG: "There are **1 open lobbies**" -> "There is **1 open lobby**"...
+  if(Number(playerCount.Open_lobbies) == 1) lobbiesText = "There is **1 open lobby**";
+  // EG: ..."1 active games" -> "1 active game."
+  if(Number(playerCount.Active_games) == 1) activeGamesText = "**1 active game.**"
+  //Removing the spaces creates left aligned text.
+/*
+There are currently 1 operatives online.           There are currently 1 operatives online.
```
Comment (Arch881010) [inline src/commands/playercount.ts:35]: Comment, explaining the removed spaces 
```js
" /n "
```

File: src/commands/schedule.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ const schedule = async (options: CommandExecuteParameters) => {
 	const cronString = dateToCron(new Date((`${date} ${time}`).trim()))
 	// create the cronjob
 	const prisma = new PrismaClient()
-	const startMessage = await channel.send(`Game time: <t:${Math.floor(dateString.getTime() / 1000)}:F>`)
+	const startMessage = await channel.send(`Game time: <t:${Math.floor(dateString.getTime() / 1000)}:F> || <t:${Math.floor(dateString.getTime() / 1000)}:R>`)
```
Comment (Arch881010) [inline src/commands/schedule.ts:77]: This adds relative time. See Github Gist [Here](https://gist.github.com/LeviSnoot/d9147767abeef2f770e9ddcd91eb85aa#discord-timestamps).

File: src/commands/playercount.ts [hunk 3]
```diff
@@ -17,7 +17,25 @@ const playerCount = async (options: CommandExecuteParameters) => {
     return isSlash ? interaction?.reply(response) : message?.reply(response)
   }
   // positive number, give more details
-  const response = `There are currently **${playerCount.Online_players} operatives online.** \n There are **${playerCount.Open_lobbies} open lobbies** and **${playerCount.Active_games} active games.**`
+
+  // Sets default text strings.
+  var operativesText = `There are currently **${playerCount.Online_players} operatives online.**`;
+  var lobbiesText = `There are **${playerCount.Open_lobbies} open lobbies**`;
+  var activeGamesText = `games`;
```
Comment (Mastagoon) [inline src/commands/playercount.ts]: I believe this should be 
```  var activeGamesText = ` ${playerCount.Active_games} games`;```

Comment (Arch881010) [reply src/commands/playercount.ts]: See [Here](https://github.com/Mastagoon/untrusted-discord/pull/8/commits/e595efb7f12d09b06d92d66645ac3505726d512b) or commit 3.

**Review by Mastagoon — APPROVED** [review]
seems good, no conflicts and builds just fine. I'm merging this

**Pull request merged**